erdpy/CLI.md

```diff
@@ -30,7 +30,7 @@ TOP-LEVEL OPTIONS:
 ----------------------
 COMMAND GROUPS summary
 ----------------------
-contract                       Build, deploy and interact with Smart Contracts
+contract                       Build, deploy, upgrade and interact with Smart Contracts
```

Comment:
Also, please edit the file `cli_contracts.py` add update the description:

```
def setup_parser(args: List[str], subparsers: Any) -> Any:
    parser = cli_shared.add_group_subparser(subparsers, "contract", "TYPE THE NEW DESCRIPTION HERE")
```

This is necessary because the file `CLI.md` is automatically generated (by actually running `erdpy help`) when invoking `CLI.md.sh`.